src/classes/GetWeather.cls

```diff
@@ -0,0 +1,108 @@
+/**
+ * Created by Mariya on 29.11.2022.
+ */
+
+public with sharing class GetWeather {
+
+    public Datetime dateAndTime {get;set;}
+    public Integer visibility  {get;set;}
+    public MainItem mainItem  {get;set;}
+    public WeatherItem weatherItem {get;set;}
+    public WindItem windItem {get;set;}
+
+    public class MainItem {
+        public Integer temp;
+        public Integer maxTemp;
+        public Integer minTemp;
+        public Integer pressure;
+        public Integer humidity;
+    }
+
+    public class WeatherItem {
+        public String descriotion;
+    }
+
+    public class WindItem {
+        public Integer windSpeed;
+    }
+
+    public static List<GetWeather> pasrseWeatherData() {
```

Comment:
MINOR
This can work but it can be done in easier way:

`public static OpenWeatherDto parseFromJSON(String body) {
      OpenWeatherDto temp = (OpenWeatherDto) JSON.deserialize(body, OpenWeatherDto.class);  
      return temp;
}`